Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -52,8 +52,21 @@ WebFoundTextRangeController::WebFoundTextRangeController(WebPage& webPage)
 {
 }
 
+#if PLATFORM(IOS_FAMILY)
+void WebFoundTextRangeController::updateFindResults()
+{
+    m_webPage->updateFindResults();
+}
+#endif
+
 void WebFoundTextRangeController::findTextRangesForStringMatches(const String& string, OptionSet<FindOptions> options, uint32_t maxMatchCount, CompletionHandler<void(Vector<WebFoundTextRange>&&)>&& completionHandler)
 {
+#if ENABLE(IMAGE_ANALYSIS) && PLATFORM(IOS_FAMILY)
+    m_webPage->corePage()->analyzeImagesForFindInPage([weakController = WeakPtr { this }] {
```

Comment:
Nit: We normally name the variable `weakThis`.

Reply:
also `weakThis = WeakPtr { *this }` (note the `*this`)